packages/shared/src/components/sidebar/SquadsList.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { ReactElement } from 'react';
+import { Squad } from '../../graphql/squads';
+import DefaultSquadIcon from '../icons/DefaultSquad';
+import NewSquadIcon from '../icons/NewSquad';
+import { SquadImage } from '../squads/SquadImage';
+import { ClickableNavItem } from './ClickableNavItem';
+import { ItemInner, NavItem, SidebarMenuItem } from './common';
+import TimerIcon from '../icons/Timer';
+
+type SquadsListProps = {
+  activePage?: string;
+  squads: Squad[];
+  onNewSquad: () => void;
+  onOpenLockedSquad: (squad: Squad) => void;
+  sidebarExpanded: boolean;
+  sidebarRendered: boolean;
+};
+
+export function SquadsList({
+  activePage,
+  squads,
+  onNewSquad,
+  onOpenLockedSquad,
+  sidebarExpanded,
+}: SquadsListProps): ReactElement {
+  const newSquadMenuItem: SidebarMenuItem = {
+    icon: () => <NewSquadIcon />,
+    title: 'New Squad',
+    action: onNewSquad,
+  };
+
+  return (
+    <>
+      {squads.map((squad) => {
+        const { handle, name, permalink, image, active } = squad;
+        const menuItem: SidebarMenuItem = {
+          icon: () =>
+            image ? (
+              <SquadImage className="w-5 h-5" {...squad} />
+            ) : (
+              <DefaultSquadIcon />
+            ),
+          rightIcon: () => sidebarExpanded && !active && <TimerIcon />,
+          title: name,
+          ...(!active
+            ? { action: () => onOpenLockedSquad(squad) }
+            : { path: `/squads/${handle}` }),
```

Comment:
Why do we provide an explicit path here? The squad object from boot provides permalink

Reply:
The permalink is an absolute path, this one is relative.

Reply:
Can we get around it somehow? I don't want to hardcode the paths

Reply:
We have other hardcoded relative paths like this. Should we change them all?
(Otherwise preview.app.daily.dev dos not work properly)

Reply:
Ok let's leave it as it for now. I'll let Chris decide in the future